etl/steps/data/meadow/war/2024-01-11/spread_of_nuclear_weapons.py

```diff
@@ -0,0 +1,65 @@
+"""Load a snapshot and create a meadow dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Retrieve snapshot.
+    snap = paths.load_snapshot("spread_of_nuclear_weapons.csv")
+
+    #
+    # Process data.
+    #
+    # The raw data is in a table in a PDF document, which is hard to parse programmatically.
+    # Instead, we manually extract the values here.
```

Comment:
Good point, it's incorrect to add the snapshot as a csv file when it's a pdf. I've corrected that. However, in this case I think it's more useful if the snapshot is the pdf, which is the real source of truth (that we can visually check), rather than the manually extracted data (which could have errors due to the manual extraction).